index.js

```diff
@@ -262,13 +260,49 @@ module.exports = {
     'selector-max-type': 2,
     'selector-max-universal': 1,
     'selector-no-qualifying-type': true,
-    'selector-no-vendor-prefix': true,
-    'shorthand-property-no-redundant-values': true,
-    'string-quotes': 'double',
     'unicode-bom': 'never',
     'value-list-comma-newline-after': 'never-multi-line',
     'value-list-comma-newline-before': 'never-multi-line',
-    'value-list-comma-space-after': 'always',
-    'value-no-vendor-prefix': true
-  }
+    'value-list-comma-space-after': 'always'
+  },
+  'overrides': [
+    {
+      'files': '**/*.scss',
+      'extends': [
+        'stylelint-config-standard-scss'
+      ],
+      'rules': {
+        'no-invalid-position-at-import-rule': null,
+        'scss/at-extend-no-missing-placeholder': null,
+        'scss/at-function-named-arguments': 'never',
+        'scss/at-if-closing-brace-newline-after': null,
+        'scss/at-if-closing-brace-space-after': null,
+        'scss/at-if-no-null': null,
+        'scss/at-mixin-argumentless-call-parentheses': 'always',
+        'scss/at-mixin-named-arguments': null,
+        'scss/at-mixin-pattern': null,
+        'scss/at-rule-conditional-no-parentheses': null,
+        'scss/comment-no-empty': null,
+        'scss/dimension-no-non-numeric-values': true,
+        'scss/dollar-variable-colon-space-after': 'at-least-one-space',
+        'scss/dollar-variable-empty-line-before': null,
+        'scss/double-slash-comment-empty-line-before': null,
+        'scss/double-slash-comment-whitespace-inside': null,
+        'scss/function-quote-no-quoted-strings-inside': null,
+        'scss/media-feature-value-dollar-variable': null,
+        'scss/no-duplicate-dollar-variables': [
+          null,
+          {
+            'ignoreInsideAtRules': [
+              'if',
+              'mixin'
+            ]
+          }
+        ],
```

Comment:
```suggestion
        'scss/no-duplicate-dollar-variables': null,
```

We don't need options here if the rule is turned off with `null`.